src/System.Private.ServiceModel/src/System/ServiceModel/Channels/HttpChannelHelpers.cs

```diff
@@ -242,8 +242,9 @@ public override Task<int> ReadAsync(byte[] buffer, int offset, int count, Cancel
             int bytesRead;
             if (ReadFromBuffer(buffer, offset, count, out bytesRead))
             {
-                return Task.FromResult<int>(bytesRead);
+                return Task.FromResult(bytesRead);
```

Comment:
It was a complaint by a code analysis tool in VS. I figured no harm in leaving a minor code cleanup. I did have other changes in this file but the final solution didn't need them.